test/index.test.ts

```diff
@@ -9,10 +9,11 @@ import { TestOptions } from './types';
 const prettierConfig = prettier.resolveConfig.sync(path.join(__dirname, '../.prettierrc'));
 
 const testCases = glob.sync('**/input.{d.ts,ts,tsx}', { absolute: true, cwd: __dirname });
+const otherFiles = glob.sync('**/external.{d.ts,ts,tsx}', { absolute: true, cwd: __dirname });
```

Comment:
Because this new test case, tests imports, we need another file name

Reply:
This is only needed for entry points, ts resolves the imports to locate the external file you added so this should be reverted

Reply:
Ah, right! Thanks!